docker/runtimes/action-nodejs-6/Dockerfile

```diff
@@ -12,7 +12,69 @@ RUN curl -L -o nodejs.tar.gz https://github.com/apache/incubator-openwhisk-runti
   && rm -rf /opt/app-root/src/nodejs/ /opt/app-root/src/nodejs.tar.gz \
   # Install
   && npm install \
+  && npm install \
```

Comment:
This is ugly, but I don't know now how to do it better.

Reply:
So running the thing twice fixes the perms issues?

Reply:
Haha, good catch, fixing :)

Reply:
Uhm, no, actually this is how it should be.